internal/config/nsmgr/namespaces.go

```diff
@@ -0,0 +1,59 @@
+// +build linux
+
+package nsmgr
+
+import (
+	"github.com/containers/storage/pkg/idtools"
+)
+
+// NSType is an abstraction about available namespace types
+type NSType string
+
+const (
+	NETNS                NSType = "net"
+	IPCNS                NSType = "ipc"
+	UTSNS                NSType = "uts"
+	USERNS               NSType = "user"
+	PIDNS                NSType = "pid"
+	ManagedNamespacesNum        = 4
+)
+
+// NamespaceIface provides a generic namespace interface
+type NamespaceIface interface {
```

Comment:
Could you rearrange the commits? I see you added code in this commit that is being dropped in the second commit.

Reply:
I believe rearranging would basically end up looking like the commits are squashed. would you just like me to do that? I left it this way to make the 1:1 relationship between this code living in nsmgr vs sandbox more clear